sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobServiceSasQueryParameters.java

```diff
@@ -240,7 +240,10 @@ UserDelegationKey userDelegationKey() {
      * Encodes all SAS query parameters into a string that can be appended to a URL.
      *
      * @return A {@code String} representing all SAS query parameters.
+     * @deprecated Please use the generate*Sas method on the desired container/blob client after initializing
```

Comment:
>@deprecated [](start = 7, length = 11)

I thought we couldn't deprecate this because it's used in BlobUrlParts

Reply:
For some reason I thought the BlobUrlParts was just initializing this class, but youre right, its using this method. Need to undeprecate (I can also add a work item to eventually get rid of this class entirely and port everything to the Util class)

Reply:
Undeprecated. Created an issue here https://github.com/Azure/azure-sdk-for-java/issues/6541